pkg/web/apps/apps_service.go

```diff
@@ -109,3 +111,90 @@ func (a *appsService) BindingAppByApp(appId, name string) error {
 	// if is deleted so just reactive the existent app
 	return a.repository.UnDeleteAppByAppID(app.AppID)
 }
+
+// // InitClientApp returns information about the current state of the app - its disabled status
+func (a *appsService) InitClientApp(deviceInfo *models.Device) (*models.Version, error) {
```

Comment:
Hi @craicoverflow,

The best way that I found to help you with it here is doing the PR: https://github.com/aerogear/mobile-security-service/pull/88/files

NOTE: 
* Now, we also need to create or update the deleted_at of the app 
* has some checks that we missed in the previous discussions too.  
* see how the constructors were made and have been used. 

I am pinging @davidffrench in order to ask for he also checks the helping/suggestion PR in order and give his thoughts.